### ISSUE TITLE
[SR-4710] swift build -Xswiftc has to be repeated

### ISSUE THREAD

**Issue by swift-ci** [apple/swift-package-manager#5031]
|                  |                 |
   |------------------|-----------------|
   |Previous ID       | SR-4710      |
   |Radar             | None         |
   |Original Reporter | amraboelela (JIRA User)      |
   |Type              | Bug    |
   |Status            | Closed        |
   |Resolution        | Won't Do    |
        
   <details>
  <summary>Additional Detail from JIRA</summary>

   |                  |                 |
   |------------------|-----------------|
   |Votes             | 0         |
   |Component/s       | Package Manager    |
   |Labels            | Bug        |
   |Assignee          | None      |
   |Priority          | Medium      |

   

   md5: a1bed3d79d6ff2f91a9a87e01788e725

  </details>

**cloned to**:  
* [SR-5851](https://bugs.swift.org/browse/SR-5851) swift build can't change -target   

**Issue Description:**

To pass 2 items to swiftc compiler currently we need to repeat -Xswiftc for each item, e.g.:  
swift build -Xswiftc -DDEBUG -Xswiftc -DAppServer

Expected behavior is to use -Xswiftc only once, like:

swift build -Xswiftc -DDEBUG -DAppServer
